.github/workflows/workflow.yml

```diff
@@ -82,6 +86,8 @@ jobs:
           tar xvfz go-ipfs.tar.gz
           sudo mv go-ipfs/ipfs /usr/local/bin/ipfs
           ipfs init
+      - name: Pipenv lock
+        run: pipenv lock
```

Comment:
think pipenv lock should be done locally and lock file added to repo. it's quite slow operation and should be done only on pipenv file changed. also, lock file guarantees that test environment will be the same. matching all packages versions

Reply:
Since these are development dependencies only we removed the Lockfile some time ago. It did only create problems and slowed down development. This step in CI is ok as most other jobs take longer. We should automate all critical tasks rather than do them locally.